### PR TITLE
fix: bun detect

### PR DESCRIPTION
**Pull request by keyding**
## Problem solved

Fix bun detection

## Changes made

- [x] Internal API changes: `packages/thirdweb/src/cli/bin.ts`

## How to test

```bash
bun packages/thirdweb/src/cli/bin.ts
```

```bash
bunx thirdweb create
```

## Contributor NFT

Paste in your wallet address below and we will airdrop you a special NFT when your pull request is merged. 

```Address: 0xC2745cd26F703C869a6d4797472f64a599C4ED65```

---

## PR-Codex overview
This PR focuses on fixing bun detection in the `bin.ts` file of the `thirdweb` package.

### Detailed summary
- Updated the `runner` variable assignment from "bun" to "bunx" for bun detection.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: packages/thirdweb/src/cli/bin.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ async function main() {
       const isBunAvailable = (() => {
         try {
           const res = execSync("bun --version", {
-            stdio: "ignore",
+            // stdio: "ignore",
```
Comment (jnsdls) [inline packages/thirdweb/src/cli/bin.ts:32]: is this part of the fix or just left over?

File: packages/thirdweb/src/cli/bin.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ async function main() {
 
       switch (true) {
         case isBunAvailable:
-          runner = "bun";
+          runner = "bunx";
```
Comment (jnsdls) [inline packages/thirdweb/src/cli/bin.ts:46]: I think this makes sense

**Review by jnsdls — COMMENTED** [review]
thanks @keyding, mainly just one quick question

File: packages/thirdweb/src/cli/bin.ts (hunk 1 above)
Comment (keyding) [reply packages/thirdweb/src/cli/bin.ts:32]: This is part of the fix, when `stdio` is set to `ignore`, `res` is `null`.

**Review by jnsdls — APPROVED** [review]
LGTM

**Pull request merged**